DataProcess.py

```
@@ -0,0 +1,217 @@
+import matplotlib.pyplot as plt
+import sympy as sp
+
+'''
+数据拟合
+'''
+class DataFit:
+    def __init__(self, x, y, degree=None):
+        self.x = np.array(x, dtype=float)
+        self.y = np.array(y, dtype=float)
+        self.n_points = len(x)
+        # 如果用户指定了阶数，就用指定的；否则走默认规则
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)